packages/sample-plugin/webpack.config.ts

```diff
@@ -20,44 +40,35 @@ const config: Configuration = {
     rules: [
       {
         test: /\.(jsx?|tsx?)$/,
-        exclude: /node_modules/,
+        exclude: /\/node_modules\//,
         use: [
           {
             loader: 'ts-loader',
             options: {
-              configFile: path.resolve(__dirname, 'tsconfig.json'),
+              configFile: pathTo('tsconfig.json'),
             },
           },
         ],
       },
       {
-        test: /\.(css)$/,
-        use: ['style-loader', 'css-loader'],
+        test: /\.(svg|png|jpg|jpeg|gif)$/,
+        type: 'asset/resource',
       },
       {
-        test: /\.(png|jpg|jpeg|gif|svg|woff2?|ttf|eot|otf)$/,
-        type: 'asset/resource',
+        test: /\.(css)$/,
+        use: ['style-loader', 'css-loader'],
```

Comment:
Note that we don't use `MiniCSSExtractPlugin` in a dynamic plugin's webpack config.

`style-loader` will embed CSS stylesheets into the JavaScript, which generally fits the idea of plugin's exposed modules (e.g. `dist/chunks/exposed-telemetryListener.js`) loaded individually and on-demand by the application.